app/models/challenge.rb

```diff
@@ -59,4 +59,8 @@ def self.all_completed
       challenge.completed?
     end
   end
+
+  def votes
```

Comment:
can we call this votes count? I feel like challenge.votes would return all the votes

Reply:
as in the vote object